docs/keto/guides/rbac.mdx

```diff
@@ -2,6 +2,234 @@
+permissions for reports.
+
+```keto-relation-tuples
+groups:finance#member@Lila
+groups:community#member@Dilan
+groups:marketing#member@Hadley
+groups:admin#member@Neel
+```
+
+## Creating Relation Tuples
+
+Let's copy all permissions we created to a `policies.rts` file with the
+following content.
+
+```keto-relation-tuples
+reports:finance#view@(groups:finance#member)
+reports:community#view@(groups:community#member)
+reports:marketing#view@(groups:marketing#member)
+reports:finance#edit@(groups:admin#member)
+reports:community#edit@(groups:admin#member)
+reports:marketing#edit@(groups:admin#member)
+reports:finance#view@(groups:admin#member)
+reports:community#view@(groups:admin#member)
+reports:marketing#view@(groups:admin#member)
+groups:finance#member@Lila
+groups:community#member@Dilan
+groups:marketing#member@Hadley
+groups:admin#member@Neel
+```
+
+
+Then we can run
+
+```bash
+keto relation-tuple parse policies.rts --format json | \
+  keto relation-tuple create - >/dev/null \
+  && echo "Successfully created tuple" \
+  || echo "Encountered error"
+```
+
+
+Since Dilan works as a community manager, the following check examples show that
+he has access only to community reports
+
+```bash
+keto check Dilan view reports finance
+Denied
+keto check Dilan view reports community
+Allowed
+keto check Dilan edit reports community
+Denied
+```
+
+Now Dilan decided to also work with marketing. Therefore we need to update his
+permissions and add him to the marketing group.
+
+```keto-relation-tuples
+groups:marketing#member@Dilan
+```
+
+Now he also has access to marketing reports:
+
+```
+keto check Dilan view reports marketing
+Allowed
+```
+
+## Display all objects a User has access to
+
+The example below shows you how to get a list of objects Dilan has access to
+
+```bash
+# Get all groups for Dilan
+curl -s http://localhost:4466/relation-tuples\?subject_id\=Dilan\&relation\=member | jq
+{
+  "relation_tuples": [
+    {
+      "namespace": "groups",
+      "object": "community",
+      "relation": "member",
+      "subject_id": "Dilan"
+    },
+    {
+      "namespace": "groups",
+      "object": "marketing",
+      "relation": "member",
+      "subject_id": "Dilan"
+    }
+  ],
+  "next_page_token": ""
+}
+
+# Get permissions to objects for marketing group
+curl -s http://localhost:4466/relation-tuples\?subject_set.namespace\=groups\&subject_set.relation\=member\&subject_set.object\=marketing | jq
+{
+  "relation_tuples": [
+    {
+      "namespace": "reports",
+      "object": "marketing",
+      "relation": "view",
+      "subject_set": {
+        "namespace": "groups",
+        "object": "marketing",
+        "relation": "member"
+      }
+    }
+  ],
+  "next_page_token": ""
+}
+# Get permissions to objects for community group
+curl -s http://localhost:4466/relation-tuples\?subject_set.namespace\=groups\&subject_set.relation\=member\&subject_set.object\=marketing | jq
```

Comment:
Here the request referenced "marketing", not "community".
```bash
keto relation-tuple get --subject-set="groups:community#member"
```

Reply:
Thanks. I've fixed it.